OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/AccelerationLimiter.mo

```diff
@@ -0,0 +1,93 @@
+    annotation (Placement(transformation(extent={{-124,32},{-92,64}}), iconTransformation(extent={{-118,38},{-92,64}})));
+  Modelica.Blocks.Interfaces.RealOutput FSRA
+    "Acceleration Controller Output"
+    annotation (Placement(transformation(extent={{100,38},{122,60}}), iconTransformation(extent={{100,38},{122,60}})));
+  Modelica.Blocks.Interfaces.RealInput FSR annotation (Placement(transformation(extent={{126,-70},{94,-38}}), iconTransformation(extent={{118,-62},{94,-38}})));
+initial equation
+  s80 = 0;
+
+equation
+  connect(Acceleration.y, add.u1) annotation (Line(
+      points={{39.2,0},{34,0},{34,0.4},{64.2,0.4}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(s8.y, add1.u2) annotation (Line(
+      points={{-48.7,-13},{-40,-13},{-40,-6},{-28,-6}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(Acceleration.u, add1.y) annotation (Line(
+      points={{11.6,0},{-5,0}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(add1.u1, ASET) annotation (Line(points={{-28,6},{-44,6},{-44,48},{-108,48}}, color={0,0,127}));
+  connect(s8.u, SPEED) annotation (Line(points={{-78.6,-13},{-87.3,-13},{-87.3,-62},{-108,-62}}, color={0,0,127}));
+  connect(add.u2, FSR) annotation (Line(points={{64.2,-10.4},{52,-10.4},{52,-54},{110,-54}}, color={0,0,127}));
+  connect(add.y, FSRA) annotation (Line(points={{84.9,-5},{90.45,-5},{90.45,49},{111,49}}, color={0,0,127}));
+  annotation (
+    Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,100}}), graphics={Text(
+                    extent={{-50,-26},{-40,-34}},
+                    lineColor={255,0,0},
+                    textString="s8")}),
+    Icon(coordinateSystem(extent={{-100,-100},{100,100}}, preserveAspectRatio=false), graphics={Rectangle(extent={{-100,100},{100,-100}}, lineColor={0,0,255}),Text(
+                    extent={{-90,62},{-58,40}},
+                    lineColor={28,108,200},
+                    textString="ASET"),Text(
+                    extent={{-62,46},{68,-46}},
+                    lineColor={28,108,200},
+                    textString="Acceleration
+Limiter"),Text(               extent={{-88,-40},{-52,-64}},
+                    lineColor={28,108,200},
+                    textString="SPEED"),Text(
+                    extent={{64,60},{96,38}},
+                    lineColor={28,108,200},
+                    textString="FSRA"),Text(
+                    extent={{68,-42},{92,-58}},
+                    lineColor={28,108,200},
+                    textString="FSR")}),
+    Documentation(info="<html>
+<table cellspacing=\"1\" cellpadding=\"1\" border=\"1\">
+<tr>
+<td><p>Reference</p></td>
+<td><p>PSS/E Manual</p></td>
+</tr>
+<tr>
+<td><p>Last update</p></td>
+<td><p>November 2020</p></td>
+</tr>
+<tr>
+<td><p>Author</p></td>
+<td><p>ALSETLab, Rensselaer Polytechnic Institute</p></td>
+</tr>
+<tr>
+<td><p>Contact</p></td>
+<td><p><a href=\"mailto:vanfrl@rpi.edu\">vanfrl@rpi.edu</a></p></td>
```

Comment:
Please point to `OpenIPSL.UsersGuide.Contact` in all places.

Reply:
Is there a model you've done this? so I can use it as a template...

Reply:
Yes, look at `OpenIPSL.Electrical.Branches.PSAT.PhaseShiftingTransformer`. Also remember that the table should go in the revison section and not the info section.